### DISCUSSION TITLE
Auto-highlight text when clicking on URL bar

### DISCUSSION THREAD

**Discussion by Anthonyy232** [zen-browser/desktop#1249] [Ideas]
As title says, it would be nice for the text to auto-highlight when clicking on the URL bar. Saves the need to do Ctrl+A before typing to clear whatever was in there already. Arc and Edge does this, and I think many other browsers do too.

**Comment by ExistentialAlex** [comment]
Either that or the copy button next to the url bar in Arc:
![image](https://github.com/user-attachments/assets/3bab2654-7410-453f-b902-276d1fbccca4)

**Comment by Anthonyy232** [comment]
Well that would be neat too, but it's a major UX pain to have to first select all the text first just to search something. Normally, browsers highlights all the texts when you click on the URL bar.

I do apologize for calling it the wrong name -- I meant auto highlight

**Reply by ExistentialAlex** [reply]
Right that makes sense. 👍🏻

**Comment by Anthonyy232** [comment]
Turns out this only occurs with Floating URLbar enabled

**Comment by ahmaadaziz** [comment]
This is a known bug and is being worked on.
For the mean time you can use CTRL + L to highlight and select the URL bar
